docs/tutorials/05_Sampling_potential_energy_surfaces.ipynb

```diff
@@ -465,16 +469,167 @@
+      "text/plain": [
+       "<Figure size 864x576 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "results_excited = {}\n",
+    "results_excited[\"points\"] = points\n",
+    "results_excited[\"energies_np\"] = np.array(res_np_exc.energies)\n",
+    "results_excited[\"energies_qeom\"] = np.array(res_qeom.energies)\n",
+    "\n",
+    "nb_excited_states = len(np.array(res_qeom.energies)[0])\n",
+    "\n",
+    "cmap = plt.cm.get_cmap(\"viridis\", len(points))\n",
+    "fig = plt.figure(figsize=(12, 8))\n",
+    "\n",
+    "for k in range(nb_excited_states):\n",
+    "    (l1,) = plt.plot(\n",
+    "        results_excited[\"points\"],\n",
+    "        results_excited[\"energies_qeom\"][:, k],\n",
+    "        linestyle=\"\",\n",
+    "        marker=\"x\",\n",
+    "        color=cmap(k / nb_excited_states),\n",
+    "    )\n",
+    "    (l2,) = plt.plot(\n",
+    "        results_excited[\"points\"],\n",
+    "        results_excited[\"energies_np\"][:, k],\n",
+    "        linestyle=\"dashed\",\n",
+    "        color=cmap(k / nb_excited_states),\n",
+    "    )\n",
+    "plt.legend([l1, l2], [\"qeom\", \"np\"])\n",
```

Comment:
In the pre-existing plots the graph for the NumPy based solver is referred to in the legend as `numpy` - perhaps we should do that here too to keep things consistent.

Reply:
Done